docs/actions.md

```diff
@@ -513,6 +514,35 @@ $ wsk action invoke --blocking --result helloPython --param name World
   }
 ```
 
+### Packaging Python actions in zip files
+
+You can package a Python action and dependant modules in a zip file.
+
+The virtual environment package *virtualenv* is supported to pass pip installed packages together with the action in the zip file.
+Package installations inside virtualenv must be done in the target environment (alpine, python 2.7), so a docker image (e.g. jfloff/alpine-python:2.7) is used to create the virtualenv directory. 
```

Comment:
Oh, now I see that you were reacting to `jfloff/alpine-python:2.7`.  I agree, it is safer to use the [pythonAction](https://github.com/openwhisk/openwhisk/blob/master/core/pythonAction/Dockerfile) image.